packages/paypal-commerce-integration/src/paypal-commerce-credit-card/paypal-commerce-credit-cards-payment-strategy.ts

```diff
@@ -421,8 +421,22 @@ export default class PayPalCommerceCreditCardsPaymentStrategy implements Payment
      * */
     private async submitHostedForm() {
         const cardFields = this.getCardFieldsOrThrow();
+        const state = this.paymentIntegrationService.getState();
+        const billingAddress = state.getBillingAddressOrThrow();
+
+        const submitConfig = {
+            billingAddress: {
+                company: billingAddress.company,
+                addressLine1: billingAddress.address1,
+                addressLine2: billingAddress.address2,
+                adminArea1: billingAddress.stateOrProvinceCode,
+                adminArea2: billingAddress.city,
+                postalCode: billingAddress.postalCode,
+                countryCode: billingAddress.countryCode,
+            },
+        };
 
-        await cardFields.submit().catch(() => {
+        await cardFields.submit(submitConfig).catch(() => {
```

Comment:
As far as i understood `try ... catch` doesn't work here as we expect?

Reply:
It should work, I just didn't refactored anything here

Reply:
Ok, leave it as it is